Dockerfile

```diff
@@ -9,6 +9,8 @@ WORKDIR $TMP_DIR
 
 COPY . .
 
+COPY config.properties /config.properties
```

Comment:
why into the root folder?

Reply:
I wanted to have a example/default file for the data2services pipeline use case. But I agree that in the root folder it is not really clean. We should put it in /example/apachedrill_config.properties